src/main/drivers/accgyro_mpu6050.c

```diff
@@ -455,3 +445,13 @@ static void mpu6050GyroRead(int16_t *gyroADC)
     gyroADC[1] = (int16_t)((buf[2] << 8) | buf[3]);
     gyroADC[2] = (int16_t)((buf[4] << 8) | buf[5]);
 }
+
+void checkMPU6050Interrupt(void) {
+	uint8_t readyForRead;
+
+	i2cRead(MPU6050_ADDRESS, MPU_RA_INT_STATUS, 1, &readyForRead);
+
+	if (readyForRead) {
```

Comment:
it would be cleaner to rename `readyForRead` to `mpuIntStatus` or similar and explicitly check `DATA_RDY_INT` bit ..